moveit_core/robot_state/src/robot_state.cpp

```diff
@@ -2315,19 +2315,19 @@ void RobotState::printTransforms(std::ostream& out) const
   const std::vector<const JointModel*>& jm = robot_model_->getJointModels();
   for (const JointModel* joint : jm)
   {
-    out << "  " << joint->getName();
+    out << ' ' << ' ' << joint->getName();
     const int idx = joint->getJointIndex();
     if (dirty_joint_transforms_[idx])
       out << " [dirty]";
-    out << ": ";
+    out << ":" << ' ';
```

Comment:
":" is still string, it should be
```suggestion
    out << ': ';
```

Reply:
:point_up:  This applies to multiple changes in this code